invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -72,6 +46,34 @@
       </div>
     {% endif %}
 
+    {% if is_preview %}
+    <div class="ui info flashed top attached manage message">
+      <div class="ui container">
+        <div class="ui relaxed grid">
+          <div class="column">
+            <div class="row">
```

Comment:
the grid is not needed here, 
to have the icon in the message it should be replaced with icon class, see https://semantic-ui.com/collections/message.html (Icon message section)